pkg/kata/proc/init.go

```diff
@@ -244,3 +246,62 @@ func (p *Init) resume(ctx context.Context) error {
 	}
 	return nil
 }
+
+// exec returns a new exec'd process
+func (p *Init) exec(context context.Context, id string, conf *ExecConfig) (Process, error) {
+	var spec specs.Process
+	if err := json.Unmarshal(conf.Spec.Value, &spec); err != nil {
+		return nil, err
+	}
+	spec.Terminal = conf.Terminal
+
+	capabilities := vc.LinuxCapabilities{
+		Bounding:    spec.Capabilities.Bounding,
+		Effective:   spec.Capabilities.Effective,
+		Inheritable: spec.Capabilities.Inheritable,
+		Permitted:   spec.Capabilities.Permitted,
+		Ambient:     spec.Capabilities.Ambient,
+	}
+
+	cmd := vc.Cmd{
+		Args:            spec.Args,
+		Envs:            []vc.EnvVar{},
+		User:            string(spec.User.UID),
+		PrimaryGroup:    string(spec.User.GID),
+		WorkDir:         spec.Cwd,
+		Capabilities:    capabilities,
+		Interactive:     spec.Terminal,
+		Detach:          !spec.Terminal,
+		NoNewPrivileges: spec.NoNewPrivileges,
+	}
+
+	_, process, err := p.sandbox.EnterContainer(p.sandbox.ID(), cmd)
```

Comment:
Same here. Please do not assume sandbox ID is the same as container ID -- there can be multiple containers in a sandbox.

Reply:
OK. I wiil modify it later.